<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="DESKTOP-ADI1JOH" failures="1" tests="1" name="Default test" time="3.197" errors="0" timestamp="2025-03-19T12:16:30 GMT">
  <testcase classname="testCases.ToDoTests" name="testE2E_EditTask" time="0.172">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index 0 out of bounds for length 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:100)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:106)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:302)
at java.base/java.util.Objects.checkIndex(Objects.java:385)
at java.base/java.util.ArrayList.get(ArrayList.java:427)
at testCases.ToDoTests.testE2E_EditTask(ToDoTests.java:111)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- testE2E_EditTask -->
  <testcase classname="testCases.ToDoTests" name="testCompleteTask" time="0.0">
    <ignored/>
  </testcase> <!-- testCompleteTask -->
  <testcase classname="testCases.ToDoTests" name="testE2E_AddMultipleTasksAndFilterActive" time="0.0">
    <ignored/>
  </testcase> <!-- testE2E_AddMultipleTasksAndFilterActive -->
  <testcase classname="testCases.ToDoTests" name="testFilterCompletedTasks" time="0.0">
    <ignored/>
  </testcase> <!-- testFilterCompletedTasks -->
  <testcase classname="testCases.ToDoTests" name="testFilterActiveTasks" time="0.0">
    <ignored/>
  </testcase> <!-- testFilterActiveTasks -->
  <testcase classname="testCases.ToDoTests" name="testAddTask" time="0.0">
    <ignored/>
  </testcase> <!-- testAddTask -->
  <testcase classname="testCases.ToDoTests" name="testDeleteTask" time="0.0">
    <ignored/>
  </testcase> <!-- testDeleteTask -->
</testsuite> <!-- Default test -->
